test/nocks.ts

```diff
@@ -13,16 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
-var nock = require('nock');
+ var nock = require('nock');
```

Comment:
This is a consequence of the auto indenting of the editor I was using.  A follow-up PR will replace this with `import * as nock from nock` and so this won't be an issue.